### DISCUSSION TITLE
gpt-neo with spacy?

### DISCUSSION THREAD

**Discussion by Huehnerbrust** [explosion/spaCy#7658] [Language Support]
Hi all,

I am just starting my space journey, and wondered if it is possible to use the new gpt-neo and later gpt-neox modells with spacy. If yes, are there any examples?

Sorry, if this is a total noob question...

Regards

**Comment by polm** [comment]
Those models have just been released a week or two, so I'm not aware of anyone using them with spaCy yet. Since there's a [HuggingFace version](https://huggingface.co/EleutherAI/gpt-neo-1.3B) of it you should be able to use it the way you would with any Transformers model - see [the docs](https://spacy.io/usage/embeddings-transformers#transformers) for an overview of that. 

That said, if you want to use GPT-Neo in the way it's typically demonstrated, as completing a prompt, that will require some work. spaCy isn't designed for text generation like that, so you'd need to write a component that sets a custom attribute or something.
